javascript/GridFieldExtensions.js

```diff
@@ -243,7 +243,11 @@
 				const classes = this.data('classes');
 				const liHtml = Object.keys(classes).map(className => {
 					const link = hrefTemplate.replace('{class}', className);
-					return `<li><a href="${link}">Add: <i>${classes[className]}</i></a></li>`;
+					const linkText = ss.i18n.inject(
```

Comment:
If we're changing this then shouldn't there be an updated bundle.js too?

Reply:
Why?

Reply:
We'll if there was a bundle.js we'd need to update it so that translations were actually used :-)

It's a non issue though

`Requirements::javascript('symbiote/silverstripe-gridfieldextensions:javascript/GridFieldExtensions.js');`